benches/benches/block_target_gas.rs

```diff
@@ -345,6 +381,97 @@ fn service_with_contract_id(
     (service, rt)
 }
 
+fn service_with_many_contracts(
```

Comment:
We have three almost similar functions to run benchmarks. Maybe we can somehow join them into one customizable?

Reply:
Did a bunch of refactoring around the services and the `run` functions.